src/parser/text_parser.mly

```diff
@@ -992,7 +992,25 @@ let inline_module_inner ==
   | fields = list(par(module_field)); {
     let fields = List.flatten fields in
     let id = None in
-    { id; fields }
+    let open Annot in
+    let open Contract in
+    let open Syntax in
+    let annots = get_annots () in
+    let annots =
+      match (list_map
+        (fun t ->
+          match t with
+          | Annot t ->
+            let+ c = parse_contract t.items in
+            Contract c
+          | _ -> Error (`Invalid_indice ""))
+      annots) with
+      | Ok annots -> annots
+      | _ -> []
```

Comment:
I'm not sure we want to ignore the error here.

Reply:
I had a look at `cmd_sym.ml` and `compile.ml`. I don't want to break into their structure too much so for now I think I'll always parse contract and generate code, and emit potential errors.